zstl/test/org/zkoss/zktest/test2/B85/B85_ZK_3794Test.scala

```diff
@@ -0,0 +1,27 @@
+package org.zkoss.zktest.test2.B85
+
+import org.junit.Test
+import org.zkoss.zstl.ZTL4ScalaTestCase
+
+/* B85_ZK_3794Test.java
+
+        Purpose:
+                
+        Description:
+                
+        History:
+                Tue Mar 13 10:46 AM:21 CST 2018, Created by klyve
+
+Copyright (C) 2018 Potix Corporation. All Rights Reserved.
+*/ class B85_ZK_3794Test extends ZTL4ScalaTestCase {
+  @Test
+  def testClick() = {
+    runZTL(
+      () => {
+        var textVal = jq(".z-textbox").`val`().trim;
+        var labelTxt = jq(".z-label:eq(1)").text().trim();
+        waitResponse()
```

Comment:
No need to `waitResponse()` here (No AJAX or animations)

Reply:
ok